modules/xmpp/ChatRoom.js

```diff
@@ -175,6 +175,7 @@ export default class ChatRoom extends Listenable {
         this.participantPropertyListener = null;
 
         this.locked = false;
+        this.transcriptionStatus = 'off';
```

Comment:
I think we should export the transcription status in the JitsiMeetJS.constants, the same way we do for recording and SIP GW. This will allow using those constants in Jitsi Meet to check the status, instead of comparing strings.